torchvision/models/detection/backbone_utils.py

```diff
@@ -105,3 +106,19 @@ def resnet_fpn_backbone(
     in_channels_list = [in_channels_stage2 * 2 ** (i - 1) for i in returned_layers]
     out_channels = 256
     return BackboneWithFPN(backbone, return_layers, in_channels_list, out_channels, extra_blocks=extra_blocks)
+
+
+def _validate_resnet_trainable_layers(pretrained, trainable_backbone_layers):
+    # dont freeze any layers if pretrained model or backbone is not used
+    if not pretrained:
+        if trainable_backbone_layers is not None:
+            warnings.warn(
+                "Changing trainable_backbone_layers has not effect if "
+                "neither pretrained nor pretrained_backbone have been set to True, "
+                "falling back to trainable_backbone_layers=5 so that all layers are trainable")
+        trainable_backbone_layers = 5
+    # by default, freeze first 2 blocks following Faster R-CNN
+    if trainable_backbone_layers is None:
```

Comment:
If someone calls pretrained=False and trainable_backbone_layers=None, the trainable layers will be 3. Is this the intended behaviour? If not, this should be an "elif".

Reply:
I believe this case would give `trainable_layers = 5` (check the indentation in line 119), can you double-check?

Reply:
You are right. Seems my "mental compiler" has a bug :P